e2e/multiple-class-psr4/expected-output.diff

```diff
@@ -0,0 +1,39 @@
+! [NOTE] File
+ !        "./multiple-class-psr4/src/C
+ !        lassMatchFileName.php" will be added
+
+ ! [NOTE] File
+ !        "./multiple-class-psr4/src/S
+ !        omeInterface.php" will be added
+
+ ! [NOTE] File
+ !        "./multiple-class-psr4/src/S
+ !        omeClass.php" will be added
+
+ ! [NOTE] File
+ !        "./multiple-class-psr4/src/S
+ !        omeOtherInterface.php" will be added
+
+ [WARNING] File "src/ClassNoMatchFileName.php" will be removed
+
+
+1 file with changes
+===================
+
+1) src/ClassMatchFileName.php:1
+
+    ---------- begin diff ----------
+@@ @@
+
+ namespace App;
+
+-class ClassMatchFileName implements SomeInterface
+-{
+-}
+-
+-interface SomeInterface
+-{
+-}
```

Comment:
This diff require improvement as it actually changed to:

```diff
+class ClassMatchFileName implements \App\SomeInterface
+{
+}
```

when not using `--dry-run`, but that will need to be in **separate PR** for improvement :)